### PR TITLE
Add `serialized_size` field to `SecureShares` in APIs 

### LINKED ISSUE #4019: Add `serialized_size` field to `SecureShares` in APIs
This will help us to know the size in bytes of Operations, Endorsements and Blocks. 

Currently, we have a method which calculates size on fly:
```rust
    /// get full serialized size
    pub fn serialized_size(&self) -> usize {
        self.serialized_data
            .len()
            .saturating_add(self.signature.get_ser_len())
            .saturating_add(self.content_creator_pub_key.get_ser_len())
    }
```

cc @longgamma-massa

### PR DESCRIPTION
**Pull request by aoudiamoncef**
* [x] document all added functions
* [ ] try in sandbox /simulation/labnet
* [x] unit tests on the added/changed features
  * [x] make tests compile
  * [x] make tests pass 
* [x] add logs allowing easy debugging in case the changes caused problems
* [x] if the API has changed, update the API specification

closes #4019

### CODE REVIEW

**Comment by aoudiamoncef** [comment]
Waiting for https://github.com/massalabs/massa/pull/4058 to be merged

**Review by bilboquet — APPROVED** [review]

File: massa-models/src/mapping_grpc.rs [hunk 1]
```diff
@@ -98,12 +102,14 @@ impl From<Endorsement> for grpc_model::Endorsement {
 
 impl From<SecureShareEndorsement> for grpc_model::SignedEndorsement {
     fn from(value: SecureShareEndorsement) -> Self {
+        let serialized_size = value.serialized_size() as u64;
```
Comment (bilboquet) [inline massa-models/src/mapping_grpc.rs:105]: you may want to inline the variable it will be more consistent with other field of the struct

Comment (aoudiamoncef) [reply massa-models/src/mapping_grpc.rs:105]: good catch... this kind of stuff  should detected by the linter

File: massa-grpc/src/stream/new_operations.rs [hunk 2]
```diff
@@ -43,24 +42,16 @@ pub(crate) async fn new_operations(
...
-                                    signature: operation.signature.to_string(),
-                                    content_creator_pub_key: operation.content_creator_pub_key.to_string(),
-                                    content_creator_address: operation.content_creator_address.to_string(),
-                                    id: operation.id.to_string()
-                                };
                                 // Send the new operation through the channel
                                 if let Err(e) = tx.send(Ok(grpc_api::NewOperationsResponse {
                                     id: request_id.clone(),
-                                    operation: Some(ret)
+                                    operation: Some(massa_operation.into())
```
[the remark this answers is not in the document]
Comment (aoudiamoncef) [reply massa-grpc/src/stream/new_operations.rs:54]: Good catch. It was I mistake

**Review by bilboquet — DISMISSED** [review]

**Comment by aoudiamoncef** [comment]
bors merge

**Comment by bilboquet** [comment]
> Build failed:
> 
>     * [build](https://github.com/massalabs/massa/actions/runs/5280726069/jobs/9554436204)

Maybe we need a training about bors? :D

**Comment by aoudiamoncef** [comment]
@bilboquet we need to reduce the time spent in CI and remove bors by the way 🥇

**Review by AurelienFT — APPROVED** [review]

**Comment by AurelienFT** [comment]
bors merge

**Review by AurelienFT — APPROVED** [review]

**Pull request merged**